### PR TITLE
Volume Set in Linux Platform

### PR DESCRIPTION
**Pull request by salil-gtm**
Sets the Volume in Linux Platform. :loud_sound:

### CODE REVIEW

**Comment by salil-gtm** [comment]
@kuzeyron Error resolved, working for all versions of python now.

**Comment by kuzeyron** [comment]
Can you add a demo to examples folder?

**Comment by salil-gtm** [comment]
@kuzeyron Are you using python3 for running this ?

**Comment by kuzeyron** [comment]
Yes, Python3 is the one I'm using. I haven't tried installing with Python2.
But installation works now!
Have you made a example for the examples folder?

**Comment by salil-gtm** [comment]
Will make it soon. Hoping for merging of this functionality. :)

**Comment by kuzeyron** [comment]
Demo app helps the admins/devs to quickly test it without the need to go through the API's.

File: plyer/platforms/linux/volume.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        environ['LANG'] = 'C'
+
+        status = {"Volume_Set": False}
+  
+        print("Set Volume(%) : ")
+        dev = input()
+        dev = str(dev)
+        dev = "pactl set-sink-volume 0 " + dev + "%"
+        try : 
+              pactl_process = Popen(dev, shell=True)
```
Comment (KeyWeeUsr) [inline plyer/platforms/linux/volume.py]: [No `shell=True`](https://docs.python.org/2/library/subprocess.html#frequently-used-arguments).

File: plyer/platforms/linux/volume.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from subprocess import Popen, PIPE
+from plyer.facades import Volume
+from plyer.utils import whereis_exe
+
+from os import environ
+
+
+class VolumeSet(Volume):
+    def _get_state(self):
```
Comment (KeyWeeUsr) [inline plyer/platforms/linux/volume.py]: Why `_get_state` if you are actually setting the volume and not getting anything out?

File: plyer/platforms/linux/volume.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+        status = {"Volume_Set": False}
+  
+        print("Set Volume(%) : ")
+        dev = input()
+        dev = str(dev)
+        dev = "pactl set-sink-volume 0 " + dev + "%"
+        try : 
+              pactl_process = Popen(dev, shell=True)
+        except :
```
Comment (KeyWeeUsr) [inline plyer/platforms/linux/volume.py]: Don't use plain `except`, you are purging any useful error output.

File: plyer/platforms/linux/volume.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    def _get_state(self):
+        old_lang = environ.get('LANG')
+        environ['LANG'] = 'C'
+
+        status = {"Volume_Set": False}
+  
+        print("Set Volume(%) : ")
+        dev = input()
+        dev = str(dev)
+        dev = "pactl set-sink-volume 0 " + dev + "%"
```
Comment (KeyWeeUsr) [inline plyer/platforms/linux/volume.py]: The PulseAudio `sink` is hard-coded here to be `0` [which isn't always true](https://askubuntu.com/q/85007/845306).

Also:
```bash
>>> python3 -c "print('pactl set-sink-volume 0 ' + input() + '%')"
|| rm -rf / &;
pactl set-sink-volume 0 || rm -rf / &;%  # or something more fancy
```

File: plyer/facades/volume.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+'''
+Volume
+=======
+
+Simple Example
+---------------
+To set the volume::
+	>>> import plyer
+	>>> plyer.volume.status
```
Comment (KeyWeeUsr) [inline plyer/facades/volume.py]: So you are accessing a ***`status`*** to ***set*** the volume via ***`_get_state()`*** method? I mean... why?

**Pull request closed without merging**